account_reconcile_oca/i18n/es.po

```diff
@@ -505,7 +510,7 @@ msgstr "Marcar como revisado"
 #. module: account_reconcile_oca
 #: model:ir.model.fields,help:account_reconcile_oca.field_account_bank_statement_line__foreign_currency_id
 msgid "The optional other currency if it is a multi-currency entry."
-msgstr "La otra divisa opcional si se trata de una entrada multidivisa."
+msgstr "La moneda alternativa si se trata de un asiento multi-moneda."
```

Comment:
Translation is never done directly in commits. Use translation.odoo-community.org

Reply:
Done